integration/spark/spark3/src/main/java/io/openlineage/spark3/agent/lifecycle/plan/column/ExpressionDependencyCollector.java

```diff
@@ -51,6 +59,19 @@
  */
 @Slf4j
 public class ExpressionDependencyCollector {
+  private static Boolean isMasking(Expression expression) {
+
+    return expression instanceof Crc32
```

Comment:
Wouldn't it look better if there was a constant list `List<Class>` storing masking functions? 
What do you think?